src/slha_format.hpp

```diff
@@ -38,6 +40,25 @@ extern const char * const single_element_formatter;
 /// SLHA line formatter for the SPINFO block entries
 extern const char * const spinfo_formatter;
 
+template <typename Container>
```

Comment:
@Expander, Dylan added extra formatting functions. You recently made a change to the way how we do it? Should we also change this?